src/main/java/com/github/manolo8/darkbot/config/Config.java

```diff
@@ -227,6 +227,7 @@
             public @Option @Num(max = 300, step = 1) int TRAIL_LENGTH = 15;
             public @Option boolean MAP_START_STOP = false;
             public @Option(key = "colors") ColorScheme cs = new ColorScheme();
+            public @Option boolean CUSTOM_BACKGROUND = false;
```

Comment:
This doesn't look like an easy way for a user to configure this custom background

Consider making it a File that the user can select via a FileChooser, or even an Image or a wrapper of an image, and an editor that essentially updates the path from a file picker

So you have essentially an object looking like:
```java
class ImageWrapper {
    private String path;
    private transient Image image;
    private transient boolean loaded;
    
    public void setPath(String path) {
        this.path = path;
        this.loaded = false;
    }
    
    public Image getImage() {
        if (loaded) return image;
        loaded = true;
        try {
            return image = ... ;
        } catch (Exception e) {
            e.printStackTrace():
            return null;
        }
    }
}
```

The editor for this type of ImageWrapper just calls setPath whenever the user selects a different path from a file picker, and then from mapDrawer you can just call getImage and if it's null do nothing